packages/gatsby-theme-docs/bin/create-release-note

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const slugify = require('slugify');
+const prompts = require('prompts');
+
+const questions = [
+  {
+    type: 'multiselect',
+    name: 'type',
+    message: 'Pick the type of the release note',
+    choices: [
+      { title: 'enhancement', value: 'enhancement' },
+      { title: 'feature', value: 'feature' },
+      { title: 'fix', value: 'fix' },
+      { title: 'announcement', value: 'announcement' },
+    ],
+  },
+  {
+    type: 'date',
+    name: 'date',
+    message: 'Pick a release date',
+  },
```

Comment:
I think it would make sense to have a default value for the "today" date.

Reply:
Better than nothing I guess if the alternative is to have to start selecting even the year from scratch.

Reply:
So at the moment the prompt has the "todays" date as default. Don't need to add an initial value here for that 🤔